book/src/developers/contributing/releases/release_checklist.md

```diff
@@ -29,17 +29,18 @@ git pull --rebase upstream master
 ```
 2. Create a new git tag with the chosen version, for example:
 ```bash
-git tag v0.1.0-alpha.15
+git tag v0.1.1
```

Comment:
I think we should still release `alpha` versions. Meaning, next release should probably be: `v0.1.1-alpha.1`.

And once we achieve milestone or want to make proper release for whatever reason, we release non-alpha (`v0.1.1`).

Reply:
I think two approaches are viable here:

- we can bump the minor version on every new release 0.2.0, 0.3.0 ... 0.123.0
- or use a pre-release identifier: 0.2.0-alpha.1 until we release a stable 0.2.0 version.

I'm more in favor of the second approach.